api/src/state.rs

```diff
@@ -165,6 +168,45 @@ impl StateApi {
             ledger_version.0,
         )
     }
+
+    /// Get raw table item
+    ///
+    /// Get a table item at a specific ledger version from the table identified by {table_handle}
+    /// in the path and the "key" (RawTableItemRequest) provided in the request body.
+    ///
+    /// The `get_raw_table_item` requires only a serialized key comparing to the full move type information
+    /// comparing to the `get_table_item` api, and can only return the query in the bcs format.
+    ///
+    /// The Aptos nodes prune account state history, via a configurable time window.
+    /// If the requested ledger version has been pruned, the server responds with a 410.
+    #[oai(
+        path = "/tables/:table_handle/raw_item",
+        method = "post",
+        operation_id = "get_raw_table_item",
+        tag = "ApiTags::Tables"
+    )]
+    async fn get_raw_table_item(
```

Comment:
This API is needed because the current api needs both the type and value information where you don't actually have the type info in `StateKey`